src/lib/Sympa/Message.pm

```diff
@@ -2167,16 +2169,22 @@ sub _append_footer_header_to_part {
         $log->syslog('debug3', "Treating text/html part");
 
         # Escape special characters.
+        #---acb: and for all three header/footer parts, wrap all URIs
+        # in HTML tags, and keep newlines for readability and for
+        # outlook.com parsing:
         $header_msg = Sympa::Tools::Text::encode_html($header_msg);
+        $header_msg = uri_finder($header_msg);
         $header_msg =~ s/(\r\n|\r|\n)$//;       # strip the last newline.
-        $header_msg =~ s,(\r\n|\r|\n),<br/>,g;
+        $header_msg =~ s,(\r\n|\r|\n),<br/>$1,g; # keep newlines
         $footer_msg = Sympa::Tools::Text::encode_html($footer_msg);
+        $footer_msg = uri_finder($footer_msg);
```

Comment:
HTML encoding (escaping) should be done only in `uri_finder()` below, i.e. `Sympa::Tools::Text::encode_html()` in above line should be removed. Or escaping will duplicate (and URIs including `&` will not be detected properly).